packages/cache/src/internal/cacheUtils.ts

```diff
@@ -71,11 +71,13 @@ export async function unlinkFile(filePath: fs.PathLike): Promise<void> {
   return util.promisify(fs.unlink)(filePath)
 }
 
-async function getVersion(app: string): Promise<string> {
-  core.debug(`Checking ${app} --version`)
+async function getVersion(app: string, args?: string[]): Promise<string> {
```

Comment:
Name args as `additionalArgs` as the primay arg is `--version` which is not being passed.